.github/workflows/pr-labeler.yaml

```diff
@@ -55,15 +55,19 @@ jobs:
       pull-requests: write
     env:
       PR_NUMBER: ${{inputs.pr-number || github.event.pull_request.number}}
+      SHELLOPTS: ${{inputs.trace && 'xtrace' || '' }}
```

Comment:
is this used anywhere?  If not please delete.

Reply:
Yes, it is. The environment variable is recognized by Bash: https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-SHELLOPTS

Here what's happening is that when the Boolean `trace` is set by using `workflow_dispatch`, it causes SHELLOPTS to get the value `xtrace`. That's the same as using `set -x`: https://www.gnu.org/software/bash/manual/bash.html#index-set

The result is that all the bash scripts in the workflow get the equivalent of `set -x`, and thus trace execution. This is usful for debugging.